server/models/Notifications.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require('mongoose');
+
+const notificationSchema = new mongoose.Schema({
+    type: {
+        required: true,
+        type: String
+    },
+    createdAt: {
+        required: true,
+        type: Date,
+        default: Date.now
+    },
+    sender: {
+        type: mongoose.Schema.Types.ObjectId,
```

Comment:
Please also add a reference to the model you would like the id's to reference. Either User or Profile? whichever way you find suitable.

Eg: `{ type: Schema.Types.ObjectId, ref: 'Person' }`